nmostesting/suites/IS0703Test.py

```diff
@@ -45,30 +45,33 @@ def set_up_tests(self):
 
         if len(self.senders) > 0:
             for sender in self.senders:
-                dest = "single/senders/" + sender + "/active/"
+                dest = "single/senders/" + sender + "/active"
                 valid, response = self.is05_utils.checkCleanRequestJSON("GET", dest)
                 if valid:
                     if len(response) > 0 and isinstance(response["transport_params"][0], dict):
                         self.sender_active_params[sender] = response["transport_params"][0]
 
     def test_01(self, test):
-        """Each Sender has ext_is_07_source_id and ext_is_07_rest_api_url parameters"""
+        """Each Sender has the required ext parameters"""
```

Comment:
Hmm, is there a bit of a future problem here? When another spec starts using WebSocket or MQTT, this test will fail because that spec's senders won't have the IS-07 `ext_` params...

Reply:
Hopefully now fixed by including IS-04 and only testing on true IS-07 resources